schema.graphql

```diff
@@ -121,3 +121,17 @@ type NativeTransfer @entity {
   transaction: Transaction!
   block: Block!
 }
+
+type Account @entity {
+  # id is the address
+  id: ID!
+  nativeBalances: [NativeBalance]! @derivedFrom(field: "account")
+}
+
+type NativeBalance @entity {
+  id: ID!
+  balanceOffset: BigInt!
+  denom: String!
```

Comment:
I think we should index this because filtering by denom will probably be a common request.

```suggestion
  denom: String! @index
```